README.md

```diff
@@ -21,15 +21,25 @@ Will return
     libelleAcheminement: 'PARIS' } ]
 ```
 
+## GéoAPI
 
-## Architecture
+Just want to quickly put together a small service? The hosted [GéoAPI](https://api.gouv.fr/api/geoapi) is a great way to build your service quickly!
 
-A source CSV file is converted to JSON and compressed on `prepublish` for faster distribution.
-On `postinstall`, it is unzipped.
-On `require`, it is indexed and loaded into memory. Finding postal codes is just an object lookup.
+However, it won't scale if you start handling lots of traffic. This module makes it easy for you to access the same database that powers the GéoAPI, on your own premises, with no centralization.
+
+Think of the GéoAPI and this `codes-postaux` module as ways to increase the use of OpenData: they are just easier paths to an open database  :)
 
 
 ## Alternatives in other languages
 
 These community-maintained libraries are based on the same dataset and offer equivalent features in other languages:
-- [Dart](https://pub.dartlang.org/packages/code_postaux)
+
+- [Dart](https://pub.dartlang.org/packages/code_postaux) by [@Kleak](https://github.com/Kleak/code_postaux).
+- Missing your favourite language? [Add your implementation](https://github.com/Kleak/sgmap/blob/master/CONTRIBUTING.md)!
```

Comment:
this link point on my github and there is no CONTRIBUTING file for the moment will put the same when it will be stable here